tools/src/commands/info_command.cc

```diff
@@ -484,9 +484,9 @@ std::tuple<double, double, double, double> InfoCommand::get_mbr(
 }
 
 // Works only for fixed-sized coordinates
-std::vector<std::string> InfoCommand::mbr_to_string(
+std::vector<std::basic_string_view<char>> InfoCommand::mbr_to_string(
```

Comment:
The passed mbr is going to have a life >= that of the returned vector, yes?
(since the returned vector will be referring to strings within that mbr...)

Reply:
Yes, but this code is also going away soon.